libs/s25main/network/GameClient.h

```diff
@@ -129,6 +130,13 @@ class GameClient final :
 
     /// Lädt ein Replay und startet dementsprechend das Spiel
     bool StartReplay(const boost::filesystem::path& path);
+
+    /// @brief Set the player infos for the AI battle.
+    /// @param playerInfos  List of AI players
```

Comment:
Maybe rather: 
```suggestion
    /// When a non-empty vector is given then an AI battle with the given AIs is started
```

as that is what it does mean. The current description is pretty much the same as the name (so the name is already very good as it doesn't need much description)

Reply:
agreed